releasenotes/notes/collect-resourceversion-31055b2a82fc885a.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Collect resourceVersion in the metadata field as well.
```

Comment:
I'm not sure it's worth a release not as it's not actionable from the user standpoint, but in any case you should probably precise the context of the feature (Kubernetes resource collection)